packages/edit/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@tailor-cms/tcee-custom-element",
```

Comment:
Since only we can use this namespace, consider omitting it. It would be nice if these would be generated based on the package name upon init.